features/autorisatie/personen/persoon/verblijfplaats/autorisatie.feature

```diff
@@ -125,12 +126,13 @@ Functionaliteit: autorisatie verblijfplaatsgegevens Persoon
       | burgerservicenummer | 000000024                                        |
       | fields              | burgerservicenummer,verblijfplaats.verblijfadres |
       Dan heeft de response de volgende gegevens
-      | naam     | waarde                                                                  |
-      | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.3             |
-      | title    | U bent niet geautoriseerd voor één of meerdere opgegeven field waarden. |
-      | status   | 403                                                                     |
-      | code     | unauthorizedField                                                       |
-      | instance | /haalcentraal/api/brp/personen                                          |
+      | naam     | waarde                                                                                                  |
+      | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.3                                             |
+      | title    | U bent niet geautoriseerd voor één of meerdere opgegeven field waarden.                                 |
+      | detail   | U bent niet geautoriseerd om de volgende gegevens op te vragen met fields: verblijfplaats.verblijfadres |
```

Comment:
hier moet verblijfplaats.verblijfadres worden hernoemd naar verblijfplaats.<ontbrekende autorisatie veld>. Dit is al gecorrigeerd na het uitvoeren van de scenarios met de gewenste implementatie